src/core/object/pipeline/regex-edits.ts

```diff
@@ -0,0 +1,18 @@
+import Song from '@/core/object/song';
+import RegexEdits from '@/core/storage/regex-edits';
+
+/**
+ * Fill song info by user defined regex edits.
+ * @param song - Song instance
+ */
+export async function process(song: Song): Promise<void> {
+	// If song info was already corrected by user, skip this stage
+	if (song.flags.isCorrectedByUser) {
+		return;
+	}
+	try {
+		await RegexEdits.loadSongInfo(song);
+	} catch (e) {
+		// Do nothing
```

Comment:
Should we log debug here to make debugging easier if something really went wrong?

Reply:
Not sure. To be honest I dont know how we would even get an error here.

But this pipeline is directly copied from pipeline/user-input, as the user-input function works basically the same due to regex edit class being directly based off the user edit model.

I assume the try catch is there for some reason, but there was no comment in the original telling me why, so I mindlessly copy pasted.

We could add some logging for sure, it shouldnt spam the user.

I’ll merge this as is for now and I can make a new PR adding some logging later, as due to the size of the changes on this PR this not being merged is blocking several future modifications needed before release.